### PR TITLE
Feature/publish

### PR DESCRIPTION
**Pull request by sayplastic**

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ var Collector = metrics.NewCollector()
 
 // Index serves a blank home page
 func Index(w http.ResponseWriter, req *http.Request) {
-	w.WriteHeader(http.StatusOK)
+	http.Redirect(w, req, "https://lbry.tv/", http.StatusSeeOther)
```
Comment (nikooo777) [inline api/handlers.go]: by hardcoding the domain here we make it impossible for the project to be served elsewhere. perhaps if we really need to pass a URL we should have it in a config file

File: app/users/authenticator.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+// Wrap result can be supplied to all functions that accept http.HandleFunc,
+// supplied function will be wrapped and called with AuthenticatedRequest instead of http.Request.
+func (a *Authenticator) Wrap(wrapped AuthenticatedFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		AccountID, err := a.GetAccountID(r)
+		ar := &AuthenticatedRequest{Request: r}
+		if err != nil {
+			ar.AuthError = err
+		} else {
+			ar.AccountID = AccountID
```
Comment (nikooo777) [inline app/users/authenticator.go]: can the accountID ever be nil or invalid? perhaps you need to check that too?

Comment (sayplastic) [reply app/users/authenticator.go]: nah, it's either retrieved from the DB or it isn't, in that case the `err` will be set.

File: api/handlers.go (hunk 1 above)
Comment (sayplastic) [reply api/handlers.go]: good point, I'll make it a setting

**Pull request merged**